hwc2_device/HwcDisplay.cpp

```diff
@@ -1083,4 +1083,19 @@ void HwcDisplay::ProcessFlatenningVsyncInternal() {
   }
 }
 
+bool HwcDisplay::GetDeviceRectSupport(hwc_rect_t display_frame) {
+  std::pair<uint32_t, uint32_t> min = pipeline_->device->GetMinResolution();
+  std::pair<uint32_t, uint32_t> max = pipeline_->device->GetMaxResolution();
```

Comment:
what's the min and max resolution is get?

Reply:
The surfaceflinger interface GetClientTargetSupport() doesn`t active, so add min/max resolution support check in hwc.
Virtiogpu min: 32x32,  max 8192x8192